app/browser/menu.js

```diff
@@ -390,9 +390,7 @@ const createBookmarksSubmenu = () => {
     },
     CommonMenu.separatorMenuItem,
     CommonMenu.bookmarksManagerMenuItem(),
-    CommonMenu.bookmarksToolbarMenuItem(),
-    CommonMenu.separatorMenuItem,
-    CommonMenu.importBookmarksMenuItem()
+    CommonMenu.bookmarksToolbarMenuItem()
   ]
```

Comment:
there is some commented out code here in the `menu.js` that I think you can delete, now that you've added import functionality. Search for `locale.translation('importFrom')`